crawler/tor.py

```diff
@@ -0,0 +1,17 @@
+import socket
+import socks
+
+
+def connectingtor():
```

Comment:
The function header could be more descriptive. Something like 
`def connect_to_tor()`

Note the underscore in the function name. (PEP 8)